GVRf/Framework/jni/contrib/assimp/include/assimp/scene.h

```diff
@@ -118,6 +118,9 @@ struct aiNode
 	  * format. Importers that don't document any metadata don't write any. 
 	  */
 	C_STRUCT aiMetadata* mMetaData;
+	
+	/* Geometric Transformations of the node */
+	aiMatrix4x4 mGeomTransformation;

```

Comment:
I have updated assimp library for the transformation fix and it is a part of that. It is not referenced in GVRf but in assimp.